src/lib/dnssd/Resolver.h

```diff
@@ -35,6 +35,14 @@
 namespace chip {
 namespace Dnssd {
 
+/// Node browsing data common to both operational and commissionable browse discovery
+struct CommonBrowseData
```

Comment:
`NodeBrowseData` seems more generic (avoiding names of common/util/helpers and similar fillters).